tests/baselines/reference/reactTagNameComponentWithPropsNoOOM.types

```diff
@@ -5,7 +5,7 @@ import * as React from "react";
 >React : typeof React
 
 declare const Tag: keyof React.ReactHTML;
->Tag : "object" | "time" | "link" | "menu" | "dialog" | "a" | "abbr" | "address" | "area" | "article" | "aside" | "audio" | "b" | "base" | "bdi" | "bdo" | "big" | "blockquote" | "body" | "br" | "button" | "canvas" | "caption" | "cite" | "code" | "col" | "colgroup" | "data" | "datalist" | "dd" | "del" | "details" | "dfn" | "div" | "dl" | "dt" | "em" | "embed" | "fieldset" | "figcaption" | "figure" | "footer" | "form" | "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "head" | "header" | "hgroup" | "hr" | "html" | "i" | "iframe" | "img" | "input" | "ins" | "kbd" | "keygen" | "label" | "legend" | "li" | "main" | "map" | "mark" | "menuitem" | "meta" | "meter" | "nav" | "noscript" | "ol" | "optgroup" | "option" | "output" | "p" | "param" | "picture" | "pre" | "progress" | "q" | "rp" | "rt" | "ruby" | "s" | "samp" | "script" | "section" | "select" | "small" | "source" | "span" | "strong" | "style" | "sub" | "summary" | "sup" | "table" | "tbody" | "td" | "textarea" | "tfoot" | "th" | "thead" | "title" | "tr" | "track" | "u" | "ul" | "var" | "video" | "wbr" | "webview"
+>Tag : keyof React.ReactHTML

```

Comment:
For all of @DanielRosenwasser's 👍 emoji's earlier, I think this is 👎 for me. We definitely want to see the shorter form in places like declaration emit and quickinfo, but not in the typewriter as it makes it harder to verify that our tests/proofs are valid.

I'd be fine with seeing the origin type if its nested inside another type (like an inferred type of `() => typeof React.ReactHTML`, but when our type writer writes the type of the declaration itself we should see the normalized type so we can verify the test proves our expectations, otherwise we can introduce regressions we'll miss due to the aliasing here.

Reply:
I guess that was @RyanCavanaugh's :+1:, not @DanielRosenwasser's, sorry.